app/components/room-card/RoomCard.types.ts

```diff
@@ -0,0 +1,3 @@
+export interface IRoomCardProps {
+  href: string;
```

Comment:
You should define everything that the card wants as props. In the next phase, you want to get data from the parent component, so you should be able to pass props from outside.

Reply:
I think you fixed that after this commit